examples/avatar-preview-app/src/PreviewApp.tsx

```diff
@@ -10,5 +10,6 @@ import { AvatarCreatorApp } from "@msquared/avatar-creator";
 
 export default function PreviewApp() {
   const dataUrl = process.env.NEXT_PUBLIC_CATALOGUE_DATA_URL || "/data.json";
-  return <AvatarCreatorApp dataUrl={dataUrl} />;
+  const animUrl = process.env.ANIM_URL || "/anim/";
```

Comment:
In production environments it's normal for assets to be hashed and have urls that aren't as simple as `/anim/idle.glb`. It would be more flexible to have a configuration object that allows specifying the absolute urls of assets that are necessary.